kotsu/error.py

```diff
@@ -0,0 +1,7 @@
+"""Exceptions and warnings."""
+
+
+class DeprecatedEntityError(ValueError):
```

Comment:
just as a style thing, for brevity for custom exceptions i think it's nicer to omit the "Error" because you typically call is like `error.ExceptionName`, or even that you are always `raise`ing or `except`ing it so it's clearly an exception. in this case you might not want to call it `DeprecatedEntity` as it sounds like an entity so maybe `EntityIsDeprecated` or something

Reply:
Not wanting to call it `DepreciatedEntity` because then it sounds like it is an Entity is exactly what my thinking was. I like `EntityIsDepreciated` ty!

Reply:
71f6671